### PR TITLE
Updated the AMA base style to properly reference presentations

### PR DESCRIPTION
**Pull request by danwchan**
`speech` is the CSL type which seems most fitting for orally presented work at a conference. `genre` holds the type presented. The AMA style guide gives examples of types as "posters" or "papers". `event`, `event-place` and `date` hold the name, location and date of the conference.

an example of a properly rendered presentation from the AMA style guide:
> Greenspan A, Eerdekens M, Mahmoud R. Is there an increased rate of cerebrovascular events among dementia patients? Poster presented at: 24th Congress of the Collegium Internationale Neuro-Psychopharmacologicum (CINP); June 20–24, 2004; Paris, France.

### CODE REVIEW

File: american-medical-association.csl [hunk 1]
```diff
@@ -215,6 +219,20 @@
         <else-if type="webpage">
           <text variable="container-title" prefix=" " suffix="."/>
         </else-if>
+        <else-if type="speech">
+          <group delimiter=" ">
+            <text variable="genre" prefix=" " suffix=" presented at:"/>
```
[the remark this answers is not in the document]
Comment (danwchan) [reply american-medical-association.csl]: Ah yes I hadn't considered the different cases of omitted information. There is no AMA recommendation in these cases but I suppose we can exercise some judgement to make it fail more gracefully.  I'll edit this with these considerations and your other suggestion regarding non-English.

[the remark this answers is not in the document]
Comment (adam3smith) [reply american-medical-association.csl]: Exactly, just make sure it looks reasonable, especially when `genre` is missing, as it often will.

**Comment by adam3smith** [comment]
Thanks -- I simplified the logic for presentations without event a bit more (note, btw., that you cannot use random terms, they need to be in the specifications)

**Comment by danwchan** [comment]
Mmm yeah I realized that as I was familiarizing myself with CSL more. The failure case where the conference name is omitted isn't graceful in my opinion. It leaves the `presented at` term hanging.

Thanks for all your help.

**Pull request merged**